README.md

```diff
@@ -0,0 +1,17 @@
+# CI Workflows
+
+This repository contains GitHub Actions workflows used by various IPLD repositories.
+By storing them in a central place (here), and distributing them in an automated way, we solve multiple problems:
+1. Consistency: Every participating repository uses the same workflow, ensuring that our code adheres to the same coding standards and is properly tested.
+2. Maintainability: Workflows change over time. We need to be able to make changes without manually updating dozens of repositories.
+
+## Technical Details
+
+This repository currently defines two workflows for Go repositories:
+* [go-check](https://github.com/ipld/.github/blob/master/workflow-templates/go-check.yml): Performs checks that ensure that the code adheres to common coding styles.
```

Comment:
all these links could be relative, like `[go-check](workflow-templates/go-check.yml)`, reducing duplication and rendering well even if this git repository gets forked or hosted elsewhere.

Reply:
Also, I think "coding styles" is too narrow; we already run vet, which finds bugs via static analysis, and we'll likely add others.

Reply:
@mvdan Any suggestion how to say this?

Reply:
@mvdan Can I ask for your help here?

Reply:
Sorry that I missed your earlier comment. How about:

> Performs static analysis, style, and formatting checks to help improve the code quality.

Reply:
Sounds good!